apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -42,6 +41,7 @@ namespace Google.Cloud.Spanner.V1
     /// </remarks>
     public sealed class PooledSession : IDisposable
     {
+        internal static readonly TransactionOptions NoTransactionOptions = new TransactionOptions();
```

Comment:
This makes me slightly nervous, because it's mutable. Internal readonly fields are just about okay if they're immutable, but it's hard to check that nothing's mutating this.

Could we make it private, but allow null in the constructors, in which case we use NoTransactionOptions (without even cloning, potentially)?